plugins/include/cstrike.inc

```diff
@@ -209,6 +209,20 @@ native cs_get_user_buyzone(index);
  */
 native cs_get_user_hasprim(index);
 
+/**
+ * Returns if the client has a weapon in the specified inventory slot.
+ *
+ * @param index     Client index
+ * @param slot      Slot to check
```

Comment:
Should we mention `CS_WEAPONSLOT_*` constants here?

Reply:
Yes.